types/src/main.ts

```diff
@@ -2389,6 +2413,14 @@ export namespace CompletionList {
 	}
 }
 
+/**
+ * @deprecated use MarkupContent instead.
+ */
+export interface MarkedStringDetail {
```

Comment:
Detail is not a good name. MarkedStringWithLanguage ?

And although deprecated we should add @since and @proposed.